Plugins/API/RainmeterAPI.h

```diff
@@ -16,6 +16,14 @@
 
 #define PLUGIN_EXPORT EXTERN_C __declspec(dllexport)
 
+struct PluginImageData
+{
+	UINT8* pixels;
+	UINT width;
+	UINT height;

```

Comment:
I consider changing them to just `int`, there might be a situation where we can't pass `uint` directly. 

For example i was trying to use stb library to load an image but the parameters only accept `int`.
So i had to use a variable of type `int` and give it to stb then assign it to the image struct.

Also i think `int` is more than enough for images, i guess no one is going to load up to 8k images or something.

Reply:
The issue with `int` isn't about which values it can take.
The issue is that we need to have a _stable_ interface that will not change when you update your compiler, or change compiler settings, or use different compiler, or something.
So, we at least need to use `int32_t` or something similar, with fixed size.
As to what exactly we should use... Um, I personally would prefer `int32_t`, but I really not sure rainmeter devs want standardized c++ types in their code, so it would be safer to use types from windows api.
"unsigned" doesn't really matter here. If somebody were to ask me, I would say that unsigned types are bullshit, unless you really-really need to use bitwise operations. But nobody is asking me, except, maybe, you. Soo, check if rainmeter devs like unsigned types, and if they are not, then I support replacing it with a signed type.

Reply:
Btw, i removed this struct completely in my mini refactor, i'm using `int32_t` for width and height, and 64 for timestamp. Also i added the `void*` in case we needed it later.